talos_rbprm/talos.py

```diff
@@ -357,12 +357,20 @@ class Robot(Parent):
     lArmx = 0.005
     lArmy = 0.005
 
+    # Paths to constraints files:
     kinematicConstraintsPath = "package://talos-rbprm/com_inequalities/"
     rLegKinematicConstraints = kinematicConstraintsPath + rleg + "_com_constraints.obj"
     lLegKinematicConstraints = kinematicConstraintsPath + lleg + "_com_constraints.obj"
     rArmKinematicConstraints = kinematicConstraintsPath + rarm + "_com_constraints.obj"
     lArmKinematicConstraints = kinematicConstraintsPath + larm + "_com_constraints.obj"
     minDist = 0.4
+    # Constraints used by SL1M:
+    filekin_right = os.environ["INSTALL_HPP_DIR"] + "/share/talos-rbprm/com_inequalities/feet_quasi_flat/COM_constraints_in_RF_effector_frame_REDUCED.obj"
```

Comment:
I don't think this environment variable is mandatory. And even if it is, we should document that a bit, as this is the first use of it in this lib

Reply:
@nim65s how would you suggest going around it ?

Reply:
This is maybe not the most elegant solution, but I think this is the simplest one:
```python
>>> from pathlib import Path
>>> import talos_rbprm
>>> def prefix(module):
...    """$prefix/lib/pythonX.Y/site-packages/$module/__init__.py: extract prefix from module"""
...    return Path(module.__file__).parent.parent.parent.parent.parent
... 
>>> prefix(talos_rbprm) / "share/talos-rbprm/com_inequalities/feet_quasi_flat/COM_constraints_in_RF_effector_frame_REDUCED.obj"
PosixPath('/opt/openrobots/share/talos-rbprm/com_inequalities/feet_quasi_flat/COM_constraints_in_RF_effector_frame_REDUCED.obj')
```

Reply:
btw, this file doesn't exist… That's `talos_COM_constraints_in_RF_effector_frame_REDUCED.obj`

Reply:
Well this is much worse no? because it assumes this is installed in /opt/openrobots/share/ ? Or am I not understanding something ?

Reply:
I didn't wrote `/opt/openrobots`, this was dynamically found by the `prefix` function here. So if you install the package somewhere else, this function will provide a different result.